photon-client/src/store/index.js

```diff
@@ -97,7 +97,7 @@ export default new Vuex.Store({
                     threads: 1,
                     debug: false,
                     refineEdges: true,
-                    numIterations: 1,
+                    numIterations: 30,
```

Comment:
What should be the default value for this?

Reply:
30 is fine. It doesn't matter, it gets overwritten by the backend.